resources/resincli.robot

```diff
@@ -40,17 +40,32 @@ Delete application "${application_name}"
 Force delete application "${application_name}"
     Run Keyword And Ignore Error    Delete application "${application_name}"
 
-Push "${git_url}":"${commit_hash}" to application "${application_name}"
-    Remove Directory    tmp    recursive=True
-    Create Directory    tmp
-    ${result} =  Run Process    git clone ${git_url} ${application_name}   shell=yes    cwd=./tmp
-    Process ${result}
+#   Push "${git_url}":"${commit_hash}" to application "${application_name}"
+#       Remove Directory    /tmp/${application_name}    recursive=True
+#       ${result} =  Run Process    git clone ${git_url} /tmp/${application_name}    shell=yes
+#       Process ${result}
+#       Set Environment Variable    RESINUSER    ${RESINUSER}
+#       ${result} =  Run Process    git remote add resin $RESINUSER@git.${RESINRC_RESIN_URL}:$RESINUSER/${application_name}.git    shell=yes    cwd=/tmp/${application_name}
+#       Process ${result}
+#       ${result} =  Run Process    git push resin ${commit_hash}:refs/heads/master    shell=yes    cwd=/tmp/${application_name}
+#       Process ${result}
+
+Get "${git_url}" to "${directory}"
```

Comment:
GIT CLONE as a keywork itself - so that the similar semantics are maintained?
`GIT CLONE ${git_url} ${directory}`